chat-plugins/uno.js

```diff
@@ -0,0 +1,563 @@
+"use strict";
+
+const maxTime = 20; // seconds
+
+function cardHTML(card, fullsize) {
+	let surface = card.value.replace(/[^A-Z0-9\+]/g, "");
+	return `<button style="height: 140px; padding-bottom: 120px; text-align: left; font-weight: bold; border: 2px solid rgba(33 , 68 , 72 , 0.59); min-width: ${fullsize ? "80" : "30"}px; border-radius: 10px 2px 2px 3px; color: ${card.colour === "Black" ? "yellow" : "black"}; background-color: ${card.colour}" name=send value="/uno play ${card.name}">${surface}</button>`;
+}
+
+function createDeck() {
+	let colours = ["Red", "Blue", "Green", "Yellow"];
+	let values = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "Reverse", "Skip", "+2"];
+
+	let basic = [];
+
+	for (let i = 0; i < 4; i++) {
+		let colour = colours[i];
+		basic.push(...values.map(v => {
+			return {value: v, colour: colour, name: colour + " " + v};
+		}));
+	}
+
+	return [...basic, ...basic, // two copies of the basic stuff (total 96)
+		...[...[...Array(4)].map((v, i) => {
+			return {colour: colours[i], value: "0", name: colours[i] + " 0"};
```

Comment:
Oh, yeah, the basic syntax of an arrow function is `input => output`.

So, for instance, a function that just returns its input is

`x => x`

This is short for `(x) => { return x; }`

With objects, you need parentheses around them so `({a: b})` so they'll be interpreted as objects rather than code blocks.